_topic_map.yml

```diff
@@ -2386,7 +2386,7 @@ Topics:
   Topics:
   - Name: Getting started with event sources
     File: knative-event-sources
-  - Name: Using the kn CLI to list event sources and event source types
+  - Name: Listing event sources and event source types
     File: serverless-kn-source
```

Comment:
I'd probably update the actual assembly name also to make it not `kn` specific since it inc. dev console now, maybe `serverless-list-source` or something?

Reply:
`serverless-listing-event-sources`